### DISCUSSION TITLE
Lettura stato allarme

### DISCUSSION THREAD

**Discussion by Nicolalabbate** [gicamm/homeassistant-comelit#67] [General]
Buongiorno, è possibile integrare l'allarme solo in lettura dello stato, senza poter inserire e disinserire l'allarme?

**Comment by gicamm** [comment]
Buongiorno,
le entita' di allarme possono essere gestite con 'alarm_control_panel'
https://www.home-assistant.io/integrations/alarm_control_panel/

dovresti lavorare su quello ed inibire armo/disarmo.
